ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -725,8 +781,8 @@ StatusDialog {
                 if (error.includes(Constants.walletSection.authenticationCanceled)) {
                     return
                 }
-                sendingError.text = error
-                return sendingError.open()
+                d.sendError = error
```

Comment:
What about the translations of errors? Shouldn't we take only e.g. type of error as enum from the backend and provide translated strings on the UI side?

Reply:
Good point, look at the line `753` and `754` what we do for router errors, we can do something similar here, but that requires some statusgo work as well. That can be one step of optimizing the sending part on the statusgo side I am planning to work on soon.

Also, just spotted, we have 2 `Connections` with the same target (lines `748` and `775`) we can remove one and just have 2 slots in a single one.